src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -148,6 +148,13 @@ export default class DefaultAudioVideoController implements AudioVideoController
     return this._mediaStreamBroker;
   }
 
+  async getRTCPeerConnectionStats(selector?: MediaStreamTrack): Promise<RTCStatsReport> {
```

Comment:
nit: `async` is not required since it doesn't `await` for any function but just return a promise.

Reply:
Removed.